### PR TITLE
New exam dialog

### PR DESCRIPTION
**Pull request by KiviKong**
#26 
Never  give you up, never  let you down
Never  run around. 

-Add the dialog window to create a new exam in the exam tab.
-Add radio buttons in the new exam and new task dialog.
-Group the radio buttons.
image

### CODE REVIEW

File: src/Controllers/CreateNewTaskDialogController.java [hunk 1]
```diff
@@ -7,21 +7,30 @@
...
+		rBtnLab.setToggleGroup(group);
+		rBtnTask.setToggleGroup(group);
 	}
 	
 	public void btnCancelAction(){
 		Stage dialogStage = (Stage) btnCancel.getScene().getWindow();		
 		dialogStage.close();
 	}
-
+
```
Comment (GalaxiasKyklos) [inline src/Controllers/CreateNewTaskDialogController.java]: Remove whitespace bitte! c:

**Comment by KiviKong** [comment]
-Change the radio buttons positions
image

**Comment by KiviKong** [comment]
- Replace the radio buttons with a choice box.
image

File: src/Controllers/CreateNewTaskDialogController.java [hunk 2]
```diff
@@ -15,17 +15,10 @@
 	
 	@FXML Button btnCancel;
 	@FXML Button btnAccept;
```
Comment (GalaxiasKyklos) [inline src/Controllers/CreateNewTaskDialogController.java]: Can you add the type choice box here as well?

Comment (GiltosMx) [reply src/Controllers/CreateNewTaskDialogController.java]: Yes, please do.

**Pull request merged**